jenkins-pipelines/diabetes_regression_ci-Jenkinsfile

```diff
@@ -16,22 +19,16 @@ pipeline {
         string(name: 'RESOURCE_GROUP', defaultValue: 'swsundar-upskill', description: 'Azure Resource Group Name')
         string(name: 'SERVICE_PRINCIPAL_NAME', defaultValue: 'swsundar-azure-spn', description: 'Azure Service Principal Name')
     }
+    environment {
+        //Pass the Jenkins' Build ID as an environment variable for tracking
+        BUILD_BUILDID = "${env.BUILD_ID}"
+    }
     stages {
-        stage('Build, Publish and Run AML Pipeline') {
-            environment {
-                BUILD_BUILDID = "${env.BUILD_ID}"
-            }
+        stage('Build and Publish AML Pipeline') {
             steps {
                 checkout([$class: 'GitSCM', branches: [[name: '*/master']],
```

Comment:
you can use "checkout scm", so it will pull source from the same branch with Jenkinsfile, and don't stick with git url and branch in the pipeline.

Reply:
Ah okay. Will update. Thanks Toan.

Reply:
Done